src/ProviderHub/help/Az.ProviderHub.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
For auto generated module, there is no need to generate help by using platyps. Just check-in the auto generated **docs** folder.